### ISSUE TITLE
How to specify br_spaces in .mdlrc for MD009

### ISSUE THREAD

**Issue by patrickmmartin** [markdownlint/markdownlint#304]
## Description
Cannot specify br_spaces in .mdlrc for MD009

### Environment
It's containerised by another team. They do not output the version. Probably recent years.

**MDL Version**
Unknown

### Expected Behavior
The trailing space parameter `br_spaces` of 2 should be allowed - this is a default for VS Code, for example and is quite popular with me.

### Actual Behavior

Any trailing spaces are flagged as a MD009 violation: looks like the parameter is not used

## Replication Case

Here is the exact .mdlrc - the `rules` line is respected, but the MD009 line is having no effect, although deliberately making the rule invalid does result in a failure to parse .mdlrc

```ruby
rules "~MD013"
rule 'MD009', :br_spaces => 2
```

It would really help to have a clearly worked through .mdlrc, with the less common cases covered explicitly, in case there are edge cases.

Thanks in advance

**Comment by jaymzh** [comment]
Rule configs go in [styles files](https://github.com/markdownlint/markdownlint/blob/master/docs/creating_styles.md) not your mdlrc.

**Comment by jaymzh** [comment]
Feel free to re-open if that doesn't answer your question

**Comment by patrickmmartin** [comment]
Ok thanks for claifying that.

**Comment by oliv3r** [comment]
@jaymzh not wanting to have to re-open this; but why? Why can I not have styles in line? I mean the 2 file choice is fine, but I really would like to only have a single file config for example. Why should that not be possible?

Edit: After some googling, it turns, you can.
```rb
rules '~MD022'
```
for example does just that. So rule**s** (plural) and negate (with a tilde) the one to be excluded. The documents subtly do refer to this as well https://github.com/markdownlint/markdownlint/blob/master/docs/configuration.md?plain=1#L63 just leaving this here for future googlers

**Comment by jaymzh** [comment]
@oliv3r yes, but you can't pass in configurations to rules, which was the original question. The configuration allows you to specify which rules to process, but configurations for those rules go in a style.
